### PR TITLE
Connect to maker behind domain

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: daemon/src/taker.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ const CONNECTION_RETRY_INTERVAL: Duration = Duration::from_secs(5);
 #[derive(Parser)]
 struct Opts {
     /// The IP address of the other party (i.e. the maker).
-    #[clap(long, default_value = "127.0.0.1:9999")]
```
Comment (bonomat) [inline daemon/src/taker.rs:34]: Imho if you remove this you will need to adapt `start-all.sh`. Also, CI smoke test might fail without a default value.

**Review by bonomat — APPROVED** [review]
Looks good. 
You can try our testnet maker under `testnet.itchysats.network:9999` :)

File: daemon/src/taker.rs (hunk 1 above)
Comment (thomaseizinger) [reply daemon/src/taker.rs:34]: Yep just saw CI failing. Will fix!

**Comment by thomaseizinger** [comment]
> Looks good. You can try our testnet maker under `testnet.itchysats.network:9999` :)

Sweet, will test against this!

Edit: Can confirm that it works.

**Review by bonomat — APPROVED** [review]
Let's go! 🚀

**Review by da-kami — APPROVED** [review]

**Pull request merged**